### ISSUE TITLE
Python 3 compatibility

### ISSUE THREAD

**Issue by nCrazed** [etianen/django-herokuapp#15]
Since both [Django 1.5](https://docs.djangoproject.com/en/1.6/topics/python3/) and [Heroku](https://devcenter.heroku.com/articles/python-runtimes#supported-runtimes) support Python 3, it would be nice if django-herokuapp did as well.

**Comment by n49ryan** [comment]
Any update on Python 3 compatibility?

**Comment by etianen** [comment]
It looks like the S3 both libraries do now support Python 3, so I guess so.

http://boto.readthedocs.org/en/latest/

I'm actually starting a new Python 3 project today that will need S3 support, so I'll see how it goes and get back to you!

**Comment by SimplyAhmazing** [comment]
Hey, I was curious if python 3 support was considered or if there were any issues that were encountered that couldn't be gotten around? Thanks. I just forked this repo and will try to convert it to py3 (where possible if it's not too difficult).

**Comment by etianen** [comment]
The issue is that django-storages currently doesn’t work with Python3.

I’ve written a replacement for Amazon S3 here: https://github.com/etianen/django-s3-storage .

There’s also an active fork of django-storages called django-storages-redux that does offer Python3 support.

Choose your poison, I guess! :P

**Comment by SimplyAhmazing** [comment]
I ran into the issue with django-storages last night when I forked this repo = pain.. I decided to not use this app just yet and the project i'm starting, doesn't need, but i really want it to use py3 :).

For now i'm going to scrape off the small things that I find helpful in this repo...

**Comment by etianen** [comment]
Absolutely. This repo is just helpers for running on Heroku. I tend to just use the README as reference when setting up my projects nowadays, as I’m now firmly in the Python3 camp!
